modules/cnf-topology-aware-lifecycle-operator-troubleshooting.adoc

```diff
@@ -0,0 +1,426 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: PROCEDURE
+[id="talo-troubleshooting_{context}"]
+= Troubleshooting the Topology Aware Lifecycle Operator
+
+The Topology Aware Lifecycle Operator is an {product-title} Operator that remediates {rh-rhacm} policies. When issues occur, use the `oc adm must-gather` command to gather details, logs and take steps in debugging the issues.
+
+For more information about related topics, see the following documentation:
+
+* link:https://access.redhat.com/articles/6218901[Red Hat Advanced Cluster Management for Kubernetes 2.4 Support Matrix]
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.0/html/troubleshooting/troubleshooting[Red Hat Advanced Cluster Management Troubleshooting]
+
+* link:https://docs.openshift.com/container-platform/latest/support/troubleshooting/troubleshooting-operator-issues.html[OpenShift Container Platform Troubleshooting Operator Issues]
+
+[id="talo-general-troubleshooting_{context}"]
+== General troubleshooting
+
+You can determine the cause of the problem by going through the following questions:
+
+* Is the configuration that you are applying supported? 
+** Are the {rh-rhacm} and the {product-title} versions compatible? 
+** Are the TALO and {rh-rhacm} versions compatible?
+* Which component is causing the problem?
+** Managed policies
+** Clusters
+** Remediation Strategy
+** TALO
+
+To ensure that the `ClusterGroupUpgrade` configuration is functional, you can do the following:
+
+. Create the `ClusterGroupUpgrade` CR with `spec.enable` set to `false`. 
+
+. Wait for the status to be updated and go through the troubleshooting questions. 
+
+. If everything looks as expected, set `spec.enable` to `true` in the `ClusterGroupUpgrade` CR.
+
+[WARNING]
+====
+After you set the `spec.enable` to `true` in the `ClusterUpgradeGroup` CR , the update procedure starts and you cannot edit the CR.
+====
+
+[id="talo-troubleshooting-modify-cgu_{context}"]
+== Cannot modify the ClusterUpgradeGroup CR
+
+Issue:: You cannot edit the `ClusterUpgradeGroup` CR after enabling the update.
+
+Resolution:: You must restart the procedure by doing the following:
```

Comment:
Small nit, remove the "You must...", "You can..." constructions here and below. Reduces word count.

```suggestion
Resolution:: Restart the procedure by doing the following:
```